backend/functions/lib/adapters/gateways/case.assignment.local.repository.ts

```diff
@@ -0,0 +1,53 @@
+import { CaseAssignmentRepositoryInterface } from '../../interfaces/case.assignment.repository.interface';
+import { CaseAttorneyAssignment } from '../types/case.attorney.assignment';
+import { ApplicationContext } from '../types/basic';
+
+export class CaseAssignmentLocalRepository implements CaseAssignmentRepositoryInterface {
+  private caseAttorneyAssignments: CaseAttorneyAssignment[] = [];
+  private _nextUnusedId = 1;
```

Comment:
We agreed to use Google's TypeScript style guide starting with this change and to leave the addition to documentation for a separate pr.

Reply:
Why would we not follow the style guide for all files? The style guide says not to use underscores in private member names and I showed you how you can set your IDE to handle this appropriately. I'm not saying that `_nextUnusedId` should be public, just that it should be named `nextUnusedId`.